packages/flutter_tools/test/general.shard/generate_localizations_test.dart

```diff
@@ -1965,7 +2018,11 @@ import 'output-localization-file_en.dart' deferred as output-localization-file_e
           throwsA(isA<L10nException>().having(
             (L10nException e) => e.message,
             'message',
-            contains('Resource attribute "@helloWorlds" was not found'),
+//             contains('''
```

Comment:
delete comment?

Reply:
Added a todo for the comment for upcoming work.